### PR TITLE
Added properties related to analytics in config rather than secret.

### PR DESCRIPTION
**Pull request by jaywalker21**
- Returning an observable in `load()` method for `DynamicConfigService` as it will be ensured that the config is loaded before anything else occurs.
- Updated `copy-secrets.sh` to only add the secret part.
- Updated `config.json` to be reflective of changes in `kube-manifests`.
- Loading analytics configuration post dynamic configuration is loaded.

### CODE REVIEW

**Review by akashmane1598 — APPROVED** [review]
LGTM

File: src/app/root.module.ts [hunk 1]
```diff
@@ -12,9 +12,11 @@ import { NavigationModule } from './shared/navigation/navigation.module';
 
 export type CustomWindow = Window &
```
Comment (akashmane1598) [inline src/app/root.module.ts:13]: @jaywalker21 Updating type of window here as well - https://github.com/razorpay/hypertrace-ui/blob/rzp_main/projects/common/src/telemetry/providers/freshpaint/load-snippet/index.d.ts#L14

**Review by akashmane1598 — COMMENTED** [review]
Left one comment

**Review by cseas — APPROVED** [review]

File: src/app/root.module.ts (hunk 1 above)
Comment (jaywalker21) [reply src/app/root.module.ts:13]: Noted, will push the change

File: projects/common/src/telemetry/providers/rudderstack/rudderstack-provider.ts [hunk 2]
```diff
@@ -5,6 +5,9 @@ import { HttpClient } from '@angular/common/http';
 import { apiObject, identify, load, page, track } from 'rudder-sdk-js';
 import { TelemetryProviderConfig, UserTelemetryProvider, UserTraits } from '../../telemetry';
 
+// tslint:disable-next-line: import-blacklist
+import type { CustomWindow } from '../../../../../../src/app/root.module';
```
Comment (jaywalker21) [inline projects/common/src/telemetry/providers/rudderstack/rudderstack-provider.ts:9]: unable to import this w/o doing `// tslint:disable-next-line`

File: src/app/root.module.ts (hunk 1 above)
Comment (jaywalker21) [reply src/app/root.module.ts:13]: Hey @akashmane1598, not going for this as this is a global declaration for `window` and don't want to mess this up.

**Review by jaywalker21 — COMMENTED** [review]
Added some changes

**Pull request merged**